syntax = "proto3";

package protocol;

message ElectionAnswer {}
